script/newsbar/rolling.js

```diff
@@ -0,0 +1,99 @@
+import { getQuerySelector, getQuerySelectorAll } from "../../utils/js/getElements.js";
+import { fetchData } from "../../utils/js/getJson.js";
+
+let intervalFirstNewsbar;
+let intervalSecondNewsbar;
+
+// 최신 헤드라인 뉴스 자동 롤링 시작
+function startRolling() {
+  intervalFirstNewsbar = setInterval(() => {
+    rollingInterval('first');
+  }, 5000);
+
+  intervalSecondNewsbar = setInterval(() => {
+    setTimeout(() => {
+      rollingInterval('second');
+    }, 1000)
+  }, 5000);
+}
+
+// 최신 헤드라인 뉴스 자동 롤링 중지
+function stopRolling() {
+  clearInterval(intervalFirstNewsbar);
+  clearInterval(intervalSecondNewsbar);
+}
+
+// 헤드라인 뉴스 자동 롤링 영역 마우스 이벤트
+function mouseEventRolling(state) {
+  const headlineNews = getQuerySelectorAll(document, `.newsbar-content-container-${state} li`);
+
+  headlineNews.forEach((elem) => {
+    elem.addEventListener('mouseover', () => {
+      elem.style.textDecoration = "underline";
+      elem.style.cursor = "pointer";
+      stopRolling();
+    })
+
+    elem.addEventListener('mouseout', () => {
+      elem.style.textDecoration = "none";
+      elem.style.cursor = "auto";
+      startRolling();
+    })
+  })
```

Comment:
이부분의 스타일을 변경하는 부분은 css에서 :hover로 따로 빼서 처리하면

```suggestion
  headlineNews.forEach((elem) => {
    elem.addEventListener('mouseover', stopRolling)
    elem.addEventListener('mouseout', startRolling)
  })
```
이렇게 깔끔해 보일 수 있을 거 같아요!!

Reply:
css자체에서도 hover을 할 때 스타일을 줄 수가 있군요,,,! 몰랐네요... 
말씀주신대로 하면 코드가 훨씬 더 깔끔해지네요 !!! 감사합니다 ~~